examples/cookbook/jotai/TodoList.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { FlatList, Pressable, Text, View } from 'react-native';
+import { atom, createStore, useAtom } from 'jotai';
+
+export type TodoItem = {
+  id: string;
+  text: string;
+};
+
+export const todosAtom = atom<TodoItem[]>([]);
+
+export function TodoList() {
+  const [todos, setTodos] = useAtom(todosAtom);
+
+  const handleAddTodo = () =>
+    setTodos((prev) => [
+      ...prev,
+      {
+        id: Math.random().toString(36).slice(2, 11),
```

Comment:
[nit] [nanoid](https://www.npmjs.com/package/nanoid) maybe?

Reply:
I've extracted this to an external `generateRandomId` to keep it tidy.